vita3k/renderer/src/state_set.cpp

```diff
@@ -312,12 +312,15 @@ COMMAND_SET_STATE(stencil_ref) {
 COMMAND_SET_STATE(fragment_texture) {
     const std::uint32_t texture_index = helper.pop<std::uint32_t>();
     SceGxmTexture texture = helper.pop<SceGxmTexture>();
-
+    auto frag_program = state->fragment_program.get(mem);
+    auto program = frag_program->program.get(mem);
+    const Sha256Hash hash_bytes = sha256(program, program->size);
```

Comment:
isn't this is hashing every texture even if texture dumping isn't enabled?
there should be no additional hashing if it's disabled

Reply:
fixed!